mc2/data/crud.py

```diff
@@ -158,7 +156,7 @@ def upload_model_data(
     dataset="analysis",
```

Comment:
add a `base_project_id="moz-fx-data-derived-datasets",` argument here that is passed to `mk_query_func(project_id=base_project_id, ...)`

Reply:
I also need the same pattern below in `print_rows_loc` to get it to run.

Reply:
I think this is an artifact of using a service account linked to `moz-fx-data-derived-datasets`, which is probably not something we want to continue doing. If using a sandbox project (or I strongly suspect, `shared-prod`), this is unnecessary.

I'm going to take this out for now, for further testing of this branch I can provision you and Saptarshi service accounts based on my test project (possibly later to be replaced by service accounts associated with a data science project).